docs/source/developer_guide/continuous_integration.rst

```diff
@@ -13,13 +13,22 @@ standards such as a common `coding style <#Coding-style>`__.
 Code quality checks
 -------------------
 
-.. _precommit: https://pre-commit.com
+.. _pre-commit: https://pre-commit.com
 
-We use `pre-commit <precommit>`_ for code quality checks.
-These checks run automatically before you make a new commit.
+We use `pre-commit`_ for code quality checks (a process we also refer to as "linting" checks).  We recommend that you set this up locally as it will
+insure that you never run into code-quality errors when you make your first PR! These checks run automatically before you make a new commit.
+To setup, first install precommit in your developer environment:
+
+::
+   pip install pre-commit
+
+Then navigate the the sktime folder and install our pre-commit configuration:
+
+::
+   pre-commit install
+
+pre-commit should now automatically run anything you make a commit! Please let us know if you have any issues with this.
```

Comment:
incomplete sentence?

I did encounter issues with GitHub desktop, which seems to get upset when `pre-commit` is on.

Reply:
an alternative would be what I've learnt from @RNKuhns recently, turning on "linting on save" in vs code

Reply:
I use Github Desktop actually, and it is fine for me.

Reply:
huh. Probably my computer is haunted then.

Reply:
I will add the linting on save in vscode.  That is a good tip. 👍